src/cfnlint/core.py

```diff
@@ -267,7 +266,14 @@ def get_template_args_rules(cli_args):
         parser.print_help()
         exit(1)
 
-    return(args, filename, template, rules, fmt, formatter)
+    return (
```

Comment:
Line 264 above needs to be changed.
`if not filename` should be `if not filenames`?